org-code-javabuilder/lib/src/main/java/org/code/neighborhood/ColorHelpers.java

```diff
@@ -7,14 +7,151 @@
+          "deeppink",
+          "palevioletred",
+          "hotpink",
+          "lightpink",
+          "pink",
+          "darkred",
+          "firebrick",
+          "crimson",
+          "indianred",
+          "lightcoral",
+          "salmon",
+          "darksalmon",
+          "lightsalmon",
+          "orangered",
+          "tomato",
+          "darkorange",
+          "coral",
+          "orange",
+          "darkkhaki",
+          "gold",
+          "khaki",
+          "peachpuff",
+          "palegoldenrod",
+          "moccasin",
+          "papayawhip",
+          "lightgoldenrodyellow",
+          "lemonchiffon",
+          "lightyellow",
+          "brown",
+          "saddlebrown",
+          "sienna",
+          "chocolate",
+          "darkgoldenrod",
+          "peru",
+          "rosybrown",
+          "goldenrod",
+          "sandybrown",
+          "tan",
+          "burlywood",
+          "wheat",
+          "navajowhite",
+          "bisque",
+          "blanchedalmond",
+          "cornsilk",
+          "darkgreen",
+          "darkolivegreen",
+          "forestgreen",
+          "seagreen",
+          "olivedrab",
+          "mediumseagreen",
+          "limegreen",
+          "springgreen",
+          "mediumspringgreen",
+          "darkseagreen",
+          "mediumaquamarine",
+          "yellowgreen",
+          "lawngreen",
+          "chartreuse",
+          "lightgreen",
+          "greenyellow",
+          "palegreen",
+          "darkcyan",
+          "lightseagreen",
+          "cadetblue",
+          "darkturquoise",
+          "mediumturquoise",
+          "turquoise",
+          "cyan",
+          "aquamarine",
+          "paleturquoise",
+          "lightcyan",
+          "darkblue",
+          "mediumblue",
+          "midnightblue",
+          "royalblue",
+          "steelblue",
+          "dodgerblue",
+          "deepskyblue",
+          "cornflowerblue",
+          "skyblue",
+          "lightskyblue",
+          "lightsteelblue",
+          "lightblue",
+          "powderblue",
+          "indigo",
+          "darkmagenta",
+          "darkviolet",
+          "darkslateblue",
+          "blueviolet",
+          "darkorchid",
+          "magenta",
+          "slateblue",
+          "mediumslateblue",
+          "mediumorchid",
+          "mediumpurple",
+          "orchid",
+          "violet",
+          "plum",
+          "thistle",
+          "lavender",
+          "mistyrose",
+          "antiquewhite",
+          "linen",
+          "beige",
+          "whitesmoke",
+          "lavenderblush",
+          "oldlace",
+          "aliceblue",
+          "seashell",
+          "ghostwhite",
+          "honeydew",
+          "floralwhite",
+          "azure",
+          "mintcream",
+          "snow",
+          "ivory",
+          "darkslategray",
+          "dimgray",
+          "slategray",
+          "lightslategray",
+          "darkgray",
+          "lightgray",
+          "gainsboro");
```

Comment:
The variety of white and offwhite colors is giving me serious flashbacks to picking a neutral color for our house.